accumulo-iterators/src/main/java/org/vertexium/accumulo/iterator/model/PropertyColumnQualifier.java

```diff
@@ -30,21 +30,34 @@ public String getDiscriminator(String visibilityString, long timestamp) {
         return getDiscriminator(getPropertyName(), getPropertyKey(), visibilityString, timestamp);
     }
 
+    public String getDiscriminator(String visibilityString) {
+        return getDiscriminatorWithoutTimestamp(getPropertyName(), getPropertyKey(), visibilityString);
+    }
+
     public static String getDiscriminator(String propertyName, String propertyKey, String visibilityString, long timestamp) {
+        String discriminatorWithoutTimestamp = getDiscriminatorWithoutTimestamp(propertyName, propertyKey, visibilityString);
```

Comment:
Can you duplicate the logic from `getDiscriminatorWithoutTimestamp` here (ie leave `getDiscriminator` as it was). `getDiscriminator` is called a lot and any performance we can squeeze out the better.